include/mpi_shared_solid_solver.h

```diff
@@ -201,7 +201,8 @@ namespace Solid
        */
       mutable std::vector<std::vector<PETScWrappers::MPI::Vector>> strain,
         stress;
-
+      //mutable PETScWrappers::MPI::Vector cellwise_sxx, cellwise_sxy,cellwise_syy;
+      mutable Vector <double> cellwise_sxx, cellwise_sxy,cellwise_syy;
```

Comment:
Should be compatible with 3D, i.e.,  when ``dim==3`` there is z direction.

Reply:
Fixed